conn_sp_test.go

```diff
@@ -94,13 +114,21 @@ func TestMoneyRead(t *testing.T) {
 
 func TestExecSpInputParams2(t *testing.T) {
 	conn := ConnectToTestDb(t)
-	err := createProcedure(conn, "test_input_params2", `
+	spBody := `
   @p1 nvarchar(255), @p2 varchar(255), @p3 nvarchar(255), @p4 nchar(10), @p5 varbinary(10), @p6 as nvarchar(255) as
   select @p1, @p2, @p3, @p4, @p5, @p6
   if exists(select * from sys.tables where name = 'tbl_test_input_params2')
      drop table tbl_test_input_params2
   select @p1 p1, @p2 p2, @p3 p3, @p4 p4, @p5 p5, @p6 p6 into tbl_test_input_params2
-  return`)
+  return`
+  	if conn.sybaseMode125() {
+  		spBody = `
+  @p1 nvarchar(255), @p2 varchar(255), @p3 nvarchar(255), @p4 nchar(20), @p5 varbinary(10), @p6 nvarchar(255) as
```

Comment:
@p4 nchar needs to be 20 in sybase 12.5 otherwise the result is only 5 characters